Makefile

```diff
@@ -0,0 +1,131 @@
+SHELL   = /bin/bash
+
+## Copyright 2016 Colin B. Macdonald
+##
+## Copying and distribution of this file, with or without modification,
+## are permitted in any medium without royalty provided the copyright
+## notice and this notice are preserved.  This file is offered as-is,
+## without any warranty.
+
+PACKAGE = $(shell grep "^Name: " DESCRIPTION | cut -f2 -d" ")
+VERSION = $(shell grep "^Version: " DESCRIPTION | cut -f2 -d" ")
+MATLAB_PACKAGE = octsympy
+BUILD_DIR = tmp
+MATLAB_PKG_DIR=${MATLAB_PACKAGE}-matlab-${VERSION}
+OCTAVE_RELEASE_TARBALL = ${BUILD_DIR}/${PACKAGE}-${VERSION}.tar
+OCTAVE_RELEASE_TARBALL_COMPRESSED = ${OCTAVE_RELEASE_TARBALL}.gz
+INSTALLED_PACKAGE = ~/octave/${PACKAGE}-${VERSION}/packinfo/DESCRIPTION
+HTML_DIR = ${BUILD_DIR}/${PACKAGE}-html
+HTML_TARBALL_COMPRESSED = ${HTML_DIR}.tar.gz
+
+OCTAVE ?= octave
+MATLAB ?= matlab
+
+.PHONY: help clean install test doctest pkg html matlab_test matlab_pkg
+
+help:
+	@echo Available rules:
+	@echo "  clean              clean all temporary files"
+	@echo "  install            install package in Octave"
+	@echo "  test               run tests with Octave"
+	@echo "  doctest            run doctests with Octave"
+	@echo "  pkg                create Octave package (${OCTAVE_RELEASE_TARBALL_COMPRESSED})"
+	@echo "  html               create Octave Forge website"
+	@echo
+	@echo "  matlab_test        run tests with Matlab"
+	@echo "  matlab_pkg         create Matlab package (${MATLAB_PKG_DIR}.zip)"
+
+
+
+${BUILD_DIR} ${BUILD_DIR}/${MATLAB_PKG_DIR}/private ${BUILD_DIR}/${MATLAB_PKG_DIR}/tests_matlab ${BUILD_DIR}/${MATLAB_PKG_DIR}/@sym ${BUILD_DIR}/${MATLAB_PKG_DIR}/@symfun ${BUILD_DIR}/${MATLAB_PKG_DIR}/@logical:
+	mkdir -p "$@"
+
+clean:
+	rm -rf "${BUILD_DIR}"
+	@#rm -f fntests.log
+	rm -f octsympy_tests.log
+
+test:
+	@echo "Testing package in GNU Octave ..."
+	@$(OCTAVE) --no-gui --silent --path "${PWD}/inst" \
+		--eval "set (0, 'defaultfigurevisible', 'off'); \
+		 octsympy_tests; \
+		 sympref reset"
+	@! grep '!!!!! test failed' octsympy_tests.log
```

Comment:
Could be!  I just copy-pasta from Doctest and/or Interval ;-)  I'll look into it.